### PR TITLE
get_iplayer: Add version 3.25.0

### PR DESCRIPTION
**Pull request by tech189**
Added [get_iplayer](https://github.com/get-iplayer/get_iplayer/)

### CODE REVIEW

File: bucket/get_iplayer.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            "get_iplayer_pvr.ico"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/get-iplayer/get_iplayer_win32/"
+    },
+    "autoupdate": {
+        "url": "https://github.com/get-iplayer/get_iplayer_win32/releases/download/$version/get_iplayer-$version-setup.exe",
+        "hash": {
+            "url": "$baseurl/SHA2-256SUMS"
```
Comment (Ash258) [inline bucket/get_iplayer.json:40]:
```suggestion
            "url": "$url.sha256"
```

File: bucket/get_iplayer.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        ],
+        [
+            "get_iplayer_web_pvr.cmd",
+            "Web PVR Manager",
+            "",
+            "get_iplayer_pvr.ico"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/get-iplayer/get_iplayer_win32/"
```
Comment (Ash258) [inline bucket/get_iplayer.json:35]:
```suggestion
        "github": "https://github.com/get-iplayer/get_iplayer_win32"
```

File: bucket/get_iplayer.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        "get_iplayer_pvr.cmd",
+        "get_iplayer_web_pvr.cmd"
+    ],
+    "shortcuts": [
+        [
+            "get_iplayer.cmd",
+            "get_iplayer",
+            "--search dontshowanymatches && cd %HOMEDRIVE%%HOMEPATH% && cmd.exe /k get_iplayer.cmd --help",
+            "get_iplayer.ico"
+        ],
```
Comment (Ash258) [inline bucket/get_iplayer.json:20]: What is this for?

Comment (tech189) [reply bucket/get_iplayer.json:20]: This is to replicate the shortcut that the installer creates at this line:

https://github.com/get-iplayer/get_iplayer_win32/blob/a1fbb3a93206240463b0392e1a0498d7f64ce7ef/get_iplayer.iss#L107

**Comment by tech189** [comment]
Thanks for correcting the above!

> Why this is not configured to application directory and persited?

Do you suggest that during the installation we should add an environment variable to change the default preferences (see [here](https://github.com/get-iplayer/get_iplayer/wiki/prefs)) folder from `%USERPROFILE%\.get_iplayer` to `~\scoop\persist\get_iplayer`?

**Comment by tech189** [comment]
@issaclin32 I do still want to merge this, just need to make another PR because this one was committed to the base branch rather than a 'patch' branch.

**Comment by issaclin32** [comment]
> @issaclin32 I do still want to merge this, just need to make another PR because this one was committed to the base branch rather than a 'patch' branch.

I have [sent a new PR](https://github.com/ScoopInstaller/Main/pull/3008), which involves the persist/profile mentioned by Ash. You can test if this works as you expected.

**Pull request closed without merging**